tests/zfs-tests/tests/functional/acl/posix/posix_001_pos.ksh

```diff
@@ -43,7 +44,14 @@
 #
 
 verify_runnable "both"
+
+function cleanup
+{
+	log_must rmdir $TESTDIR/dir.0
```

Comment:
nit: we probably don't want to `log_must` here, just `rmdir`, since the directory might not exist if one of the earlier failure occurs.